src/gtimelog/ui/components/windows.py

```diff
@@ -0,0 +1,767 @@
+import datetime
+import functools
+import os
+import smtplib
+from contextlib import closing
+from email.utils import parseaddr
+from gettext import gettext as _
+from gi.repository import Gtk, Gdk, GLib, Gio, GObject, Soup
```

Comment:
There should be a blank line between standard library imports and 3rd-party module imports, as per PEP-8.

(I really need to get that isort configuration into git master)

Reply:
Applied